applications/script.js

```diff
@@ -313,42 +317,64 @@ function createApplicationCard({ application }) {
     innerText: application.intro.introduction.slice(0, 200),
   });
 
-  const viewDetailsButton = createElement({
-    type: 'button',
-    attributes: { class: 'view-details-button' },
-    innerText: 'View Details',
-  });
-
-  viewDetailsButton.addEventListener('click', () => {
-    addQueryParamInUrl('id', application.id);
-    openApplicationDetails(application);
-  });
-
   applicationCard.appendChild(userInfoContainer);
   applicationCard.appendChild(introductionText);
-  applicationCard.appendChild(viewDetailsButton);
+
+  if (dev) {
+    applicationCard.style.cursor = 'pointer';
+    applicationCard.addEventListener('click', () => {
+      addQueryParamInUrl('id', application.id);
+      openApplicationDetails(application);
+    });
+  } else {
+    const viewDetailsButton = createElement({
+      type: 'button',
+      attributes: { class: 'view-details-button' },
+      innerText: 'View Details',
+    });
+
+    viewDetailsButton.addEventListener('click', () => {
+      addQueryParamInUrl('id', application.id);
+      openApplicationDetails(application);
+    });
+    applicationCard.appendChild(viewDetailsButton);
+  }
 
   return applicationCard;
 }
 
-async function renderApplicationCards(next, status, isInitialRender) {
+function updateTotalCount(total, status) {
+  if (total > 0) {
```

Comment:
Do we have test for totalCountElement?

Reply:
Yes there is a test that checks the existence of totalCountElement